Packs/CortexXDR/IncidentFields/XDR_Status_v2.json

```diff
@@ -28,7 +28,8 @@
 	"selectValues": [
         "new",
         "under_investigation",
-		"resolved_threat_handled",
+		"resolved_true_positive",
+		"resolved_security_testing",
```

Comment:
[in page 38 ](https://docs.paloaltonetworks.com/content/dam/techdocs/en_US/pdf/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api.pdf) you can see that the RESOLVED_THREAT_HANDLED was replaced with RESOLVED_TRUE_POSITIVE and RESOLVED_SECURITY_TESTING was added

Reply:
stays with the original value